src/metabase/models/serialization.clj

```diff
@@ -1548,11 +1562,25 @@
   [k xform]
   (assoc xform :as k))
 
+(defn- compose*
+  "Given two functions that transform the value at `k` within `x`, return their composition."
+  [f g]
+  (fn [m k x]
```

Comment:
Currently this is the only time a transform function uses `k`. Perhaps that's a smell, and `k` should rather be an argument to `compose`. I prefer it this way, but its worth discussing.

Reply:
Yeah I think even if we don't use `k` much, it's quite useful for generic transforms - and maybe we'll have time to do them at some point.